### ISSUE TITLE
Description in other language not shown

### ISSUE THREAD

**Issue by andrixnet** [opencaching/opencaching-pl#575]
http://www.opencaching.ro/viewcache.php?cacheid=16

Description is in 2 languages: RO and EN. 
While viewing the website in RO language, and this cache, the RO description is shown.
![desc1](https://cloud.githubusercontent.com/assets/3382777/14811204/9fc75434-0b9f-11e6-96cf-772b7d600d72.png)

Click on shown link to show description, while website is in RO language, page is reloaded but still RO description is shown for : 
http://www.opencaching.ro/viewcache.php?cacheid=16&desclang=EN
![desc2](https://cloud.githubusercontent.com/assets/3382777/14811218/b427a37a-0b9f-11e6-8a43-72e79f0e2906.png)

If I switch website to EN language, for this cache, the EN language is shown. 
![desc3](https://cloud.githubusercontent.com/assets/3382777/14811213/aea942e6-0b9f-11e6-80a2-14aef0d26955.png)

---

Note: since automatic translations with Google became unavailable, because that service is no longer available for free, config has this:

```
// set this to true to disable automatic translation of cache descs
$disable_google_translation = true;
```

**Comment by harrieklomp** [comment]
Why not remove the link in description language choice? If there are different descriptions like RO and EN just change the site language.

Or am i wrong?

**Comment by andrixnet** [comment]
The descriptions may differ (not 100% translations of each other). 
I believe it is useful to have access to all.

**Comment by harrieklomp** [comment]
If the cache owner does make 2 or 3 translations he is responsible for correct translations.

**Comment by andrixnet** [comment]
Owner may also choose to put somewhat different description in other language, perhaps shorter, simpler. Think of how Wikipedia articles are in different languages. 

Also, viewer may want to view different versions of the description without changing the website interface language (which is remembered)

**Comment by andrixnet** [comment]
Description may also be available in other language then website interface languages.

**Comment by andrixnet** [comment]
Apparently if the website current language is OTHER then default language, the feature works. (eg. lang=en on OCRO)
If website current language is default language, the feature does not work (ie. lang=ro on OCRO)

**Comment by andrixnet** [comment]
There are new reports about this same problem.

**Comment by andrixnet** [comment]
Issue has been solved, though no reference made here. Closing.

**Comment by deg-pl** [comment]
I've fixed it here d5c5585a1ada9ccdd5218b8a23e4db095699580d
